crates/burn-jit/src/template/base.rs

```diff
@@ -19,12 +19,13 @@ pub struct SourceKernel<K> {
 }
 
 impl<C: Compiler, K: KernelSource> CubeTask<C> for SourceKernel<K> {
-    fn compile(&self, _mode: ExecutionMode) -> CompiledKernel<C> {
+    fn compile(&self, _options: &C::CompilationOptions, _mode: ExecutionMode) -> CompiledKernel<C> {
         let source_template = self.kernel_source.source();
         let source = source_template.complete();
 
         CompiledKernel {
-            name: Some(core::any::type_name::<K>()),
+            entrypoint_name: "kernel".to_string(),
```

Comment:
Why change entry point name?

Reply:
It's an update to cubecl, `name` is now `debug_name` and there's a new `entrypoint_name`. However, it should be "main" and not "kernel", I'm fixing that in the PR I'm currently opening.